zerver/webhooks/buildbot/doc.md

```diff
@@ -0,0 +1,11 @@
+Zulip supports integration with CircleCI and can notify you of
+your build statuses.
+
+1. {!create-stream.md!}
+
+2. {!create-bot-construct-url-indented.md!}
```

Comment:
I would make all the numbers here 1, the Markdown preprocessor automatically handles the numbering.